content/contact.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Contact"
+date: 2017-07-03T14:28:34-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: ""
+projectname: ""
+listdescription: ""
+---
+
+### Contact
+If you would like to reach out to the NASA WorldWind team, please try the following:
+
+---
+#### NASA Program Management
+
+The WorldWind project is managed from the NASA Ames center in Sunnyvale, CA. The government lead and point of contact:
+
+##### Patrick Hogan - [patrick.hogan@nasa.gov](mailto://patrick.hogan@nasa.gov)
+
+---
+
+#### Issue Reports and Source Code
+WorldWind uses [GitHub](https://github.com/NASAWorldWind/) for source and issue management. Please head on over to check out the latest code or report issues or requests.
+
+---
+
+#### Development Community
+The [WorldWind Forum](https://forum.worldwindcentral.com/) is a great resource to learn more about the WorldWind projects. Many users are quick to help and share code snippets.
```

Comment:
In this case the additional text is really helpful. I think it provides a better impression of what kind of help can be found on the forum.